CMakeLists.txt

```diff
@@ -27,123 +30,136 @@ if (ENABLE_DEVELOPER_DEFAULTS)
   set(ENABLE_TOOLS On CACHE BOOL "")
 endif ()
 
-set(ENABLE_CUDA Off CACHE BOOL "")
-set(ENABLE_HIP Off CACHE BOOL "")
-set(ENABLE_SYCL Off CACHE BOOL "")
-set(ENABLE_NUMA Off CACHE BOOL "")
-set(ENABLE_OPENMP Off CACHE BOOL "")
-set(ENABLE_COPY_HEADERS Off CACHE BOOL "")
-set(ENABLE_GMOCK On CACHE BOOL "")
-set(ENABLE_CLANGQUERY Off CACHE BOOL "")
-
-option(ENABLE_LOGGING "Build Umpire with Logging enabled" On)
-option(ENABLE_SLIC "Build Umpire with SLIC logging" Off)
-option(ENABLE_BACKTRACE "Build Umpire with allocation backtrace enabled" Off)
-option(ENABLE_BACKTRACE_SYMBOLS "Build Umpire with symbol support" Off)
-option(ENABLE_COVERAGE "Enable code coverage (with GCC)" Off)
-option(ENABLE_PEDANTIC_WARNINGS "Enable pedantic compiler warnings" On)
-option(ENABLE_INACCESSIBILITY_TESTS "Test allocator inaccessibility functionality" Off)
-option(ENABLE_TOOLS "Enable Umpire development tools" Off)
-option(ENABLE_DEVICE_CONST "Enable constant memory on GPUs" Off)
-option(ENABLE_OPENMP_TARGET "Enable OpenMP target support" Off)
-option(ENABLE_PERFORMANCE_TESTS "Enable additional performance tests" Off)
-option(ENABLE_ASAN "Enable use with address sanitizer tools" Off)
-option(ENABLE_SANITIZER_TESTS "Enable address sanitizer tests" Off)
+set(UMPIRE_ENABLE_SYCL Off CACHE BOOL "")
+set(UMPIRE_ENABLE_NUMA Off CACHE BOOL "")
+set(UMPIRE_ENABLE_COPY_HEADERS Off CACHE BOOL "")
```

Comment:
This option is very very old and can actually be removed completely.

Reply:
Done